api/VRDisplay.json

```diff
@@ -1051,54 +1051,6 @@
           }
         }
       },
-      "hardwareUnitId": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/VRDisplay/hardwareUnitId",
```

Comment:
I doubt anyone is willing to add anything for deprecated WebVR, what do you think?

Reply:
I tried to remove this entry but got stuck cleaning up MDN. It's referenced in examples that I didn't know how to rewrite.